src/factories/rectangle.ts

```diff
@@ -0,0 +1,22 @@
+import { useSubscription } from '@prefecthq/vue-compositions'
+import { Graphics, RenderTexture, Sprite } from 'pixi.js'
+import { waitForApplication } from '@/objects'
+
+async function rectangle(): Promise<RenderTexture> {
+  const application = await waitForApplication()
+
+  const rectangle = new Graphics()
+  rectangle.beginFill('#fff')
+  rectangle.drawRect(0, 0, 1, 1)
```

Comment:
Since this creates a bitmap, then stretches it wide/tall enough to cover the geometry it's applied to, 1px might be a bit aggressive? You can imagine if it needs to stretch over a 500px wide thing, it'll need to scale to something well into the 10s of thousands to cover it, whereas a 10px x 10px even gives more solid room to chew on? I don't know how much it theoretically matters though.

Reply:
Since it has hard edges I don't think it matters? But I have no preference here. Just seemed like the most flexible and smallest texture. Tbh I kinda feel like we shouldn't need this but didn't want to stray to far from an implementation we know works.

Reply:
ah you might be right on the square.. At some point in the past I was able to see the texture applied to the geometry in the pixi dev tools, but can't seem to figure out how I did that. Seems to be working just fine as is.

Reply:
Ah you might be right on the square thing.. At some point I was able to see the texture scaled in the pixi dev tools, but not able to now. Looks great locally so nbd